test/test_helper.rb

```diff
@@ -0,0 +1,191 @@
+require 'active_support/core_ext/string/inflections'
+require 'factory_girl'
+require 'faker'
+require 'simplecov'
+require 'test/unit'
+require 'webmock/test_unit'
+
+require File.join(File.dirname(__FILE__), 'factories')
+
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+$LOAD_PATH.unshift(File.dirname(__FILE__))
+require 'blockscore'
+
+URL_REGEXP = %r([\w\d]*:@api\.blockscore\.com(:443)?/(?<resource>\w+)/(?<id>[\w\d]*)?/?(?<action>hits|history|score)?)
+
+WebMock.disable_net_connect!(:allow => 'codeclimate.com')
+
+HEADERS = {
+  'Accept' => 'application/vnd.blockscore+json;version=4',
+  'User-Agent' => 'blockscore-ruby/4.1.0 (https://github.com/BlockScore/blockscore-ruby)',
+  'Content-Type' => 'application/json'
+}
+
+def index_response(resource, count)
+  {
+    :total_count => count,
+    :has_more => false,
+    :object => 'list',
+    :data => FactoryGirl.build_list(resource.to_sym, count)
+  }.to_json
+end
+
+def resource_from_uri(uri)
+  uri.to_s.split('/').last.singularize.to_sym
+end
+
+# configure test-unit for FactoryGirl
+class Test::Unit::TestCase
+  include FactoryGirl::Syntax::Methods
+
+  setup do
+
+    stub_request(:any, /.*api\.blockscore\.com\/.*/).
+      with(headers: HEADERS).
+      to_return do |request|
+        match = URL_REGEXP.match request.uri.to_s
```

Comment:
This regex is pretty unwieldy. I know I recommended throwing a regex in here, but now that I think about it I think it would make more sense to use URI.  Something like

```ruby
resource, id, action = request.uri.path.split('/').tap(&:shift)
```